PowerEditor/src/WinControls/FileBrowser/fileBrowser.h

```diff
@@ -188,6 +189,25 @@ class FileBrowser : public DockingDlgInterface {
 
 	bool selectCurrentEditingFile() const;
 
+	struct FilesToChange {
+		generic_string commonPath; // Common path between all the files. rootPath + linarWithoutLastPathElement
+		generic_string rootPath;
+		std::vector<generic_string> linarWithoutLastPathElement;
+		std::vector<generic_string> files; // file/folder names
+	};

```

Comment:
The member variable of class or struct should have the prefix `_`.
(Voir https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#member-variables)

Reply:
Done